### DISCUSSION TITLE
How to install material-react-table from Github source when using npm?

### DISCUSSION THREAD

**Discussion by aram0112** [KevinVandy/material-react-table#472] [Q&A]
I noticed that this repository seems to be using PNPM. I want to install the package from source in order to gain access to a fix merged three days ago regarding renderRowActions and stale state. However, presumably since I am using npm, I am running into errors when trying to install this package via the Github URL, does anybody have any guidance or a workaround for this, or do I need to wait for the next NPM version to be uploaded. Thank you!

**Comment by KevinVandy** [comment]
the update will come, I just didn't get to it. I don't recommend ever installing libraries like that

**Reply by KevinVandy** [reply]
published v1.9.4. Fixed the row action menu items problem of the same sort too

https://www.material-react-table.dev/?path=/story/fixed-bugs-useeffects--render-row-actions-async
https://www.material-react-table.dev/?path=/story/fixed-bugs-useeffects--render-row-action-menu-items-async
